### PR TITLE
Log Colors

### PR DESCRIPTION
**Pull request by tristantarrant**

### CODE REVIEW

File: server/tests/src/test/resources/configuration/log4j2.xml [hunk 1]
```diff
@@ -71,7 +71,9 @@
 
     <Logger name="org.jgroups" level="INFO"/>
 
-    <Logger name="tc" level="DEBUG"/>
+    <Logger name="tc" level="INFO"/>
+
+    <Logger name="okhttp3.OkHttpClient" level="DEBUG"/>
```
Comment (wburns) [inline server/tests/src/test/resources/configuration/log4j2.xml:76]: Are the changes in this file required? It seems tangentially related.

Comment (tristantarrant) [reply server/tests/src/test/resources/configuration/log4j2.xml:76]: Only the `tc` one is desired. We definitely don't want okhttp debug

**Comment by tristantarrant** [comment]
Fixed

**Comment by ryanemerson** [comment]
Thanks @tristantarrant

**Pull request merged**